src/cursor/cur_std.c

```diff
@@ -751,11 +751,16 @@ __wt_cursor_cache_get(WT_SESSION_IMPL *session, const char *uri,
 			return (WT_NOTFOUND);
 	}
 
+	if (to_dup != NULL) {
+		uri = to_dup->uri;
+		hash_value = to_dup->uri_hash;
```

Comment:
I'd add a comment that we expect the caller to check that one of `uri` or `to_dup` is NULL.